src/guiguts/tools/pptxt.py

```diff
@@ -0,0 +1,2591 @@
+        s[53] = ["vou", "voung", "vour", "vouth", "vovage", "vowcl", "voyagc", "waa"]
+        s[54] = ["wagc", "waiking", "waitcr", "wakc", "wam", "wandcr", "warinth"]
+        s[55] = ["warmtb", "warmtli", "warrn", "warrnth", "waru", "wasb", "wasli"]
+        s[56] = ["wasnt", "wastc", "watcb", "watcli", "watcr", "wauder", "waut", "wav"]
+        s[57] = ["wavc", "wbat", "Wbat", "wbeat", "wbeel", "wben", "Wben", "Wbere"]
+        s[58] = ["wbere", "Wbeu", "wbich", "wbile", "Wbile", "wbilst", "wbip", "wbite"]
+        s[59] = ["wbo", "Wbo", "wbole", "wbom", "wbose", "wby", "Wc", "wc", "wcak"]
+        s[60] = ["wcakcn", "wcalth", "wcapon", "wcar", "wcavc", "wccd", "wcck", "wcigh"]
+        s[61] = ["wcight", "wcll", "wcst", "wct", "weakeu", "wealtb", "weapou", "weigb"]
+        s[62] = ["weigbt", "welcorne", "westem", "whcat", "whccl", "whcn", "Whcn"]
+        s[63] = ["Whcrc", "whcrc", "Whcre", "Wherc", "wheu", "Wheu", "whicb", "Whicb"]
+        s[64] = ["Whicb", "whicli", "whilc", "Whilc", "whitc", "whitcn", "whiteu"]
+        s[65] = ["whoin", "wholc", "whorn", "whosc", "whv", "wickcd", "widc", "widcly"]
+        s[66] = ["widcn", "widelv", "wideu", "widtb", "widtli", "wifc", "winc"]
+        s[67] = ["winncr", "wintcr", "wipc", "wirc", "wisb", "wisc", "wisdoin"]
+        s[68] = ["wisdorn", "wisli", "Witb", "witb", "witbin", "witbout", "withiu"]
+        s[69] = ["withm", "witli", "Witli", "witliin", "wiu", "wiud", "wiudow", "wiue"]
+        s[70] = ["wiug", "wiuter", "wiuuer", "Wliat", "wliat", "wlien", "Wlien"]
+        s[71] = ["Wliere", "wliere", "wliich", "Wliile", "wliile", "wliilst", "wliip"]
+        s[72] = ["wliite", "Wlio", "wlio", "wliole", "wliom", "wliose", "wliy", "wmd"]
+        s[73] = ["wmdow", "wme", "wmg", "wmner", "wmter", "wo", "woinan", "womau"]
+        s[74] = ["wondcr", "wonld", "wonnd", "woodcn", "woodeu", "woolcn", "wooleu"]
+        s[75] = ["worin", "workcr", "wornan", "worrn", "worrv", "worsc", "wortb"]
+        s[76] = ["wortli", "wouder", "wouid", "wouldnt", "wouud", "wrcck", "writc"]
+        s[77] = ["writcr", "wroug", "ycar", "ycllow", "ycs", "yct", "yicld", "yntir"]
+        s[78] = ["yonng", "yonr", "yonth", "youtb", "youtli", "youug", "youve", "zcro"]
+        s[79] = ["stiil", "aword", "nnd", "KEW", "Sonth", "wa", "ou", "aa", "klnd"]
+        s[80] = ["tne", "ths"]
+        for i in range(len(s)):
+            scannos_list = s[i]
+            for j in range(len(scannos_list)):
+                scannos_misspelled.append(scannos_list[j])
+        s = {}
+
+        # Can add new entries above. Maintain the same code structure with no line longer
+        # than 88 characters and no dictionary index greater than 99. Note the 's = {}'
+        # line above is superfluous there but is REQUIRED if new dictionary entries are
+        # added after loop-code (see above) that appends dictionary entries to the list
+        # misspelled_scannos.
```

Comment:
@G4OEU, could you explain the reasoning behind building the dictionary in this way via the small temporary dictionaries, to help decide if it's the best way to do that.

Reply:
The use of small temporary dictionaries is no longer needed. I used them because I misunderstood a _flake8_ error message. I had originally arranged four scannos on each line so that they were aligned vertically into four neat columns. This meant multiple spaces after commas but _flake8_ objected to that with an "E241 multiple spaces after ','" message for each line but the last. I later discovered there is a way to tell _flake8_ to ignore that issue. However it is better to simply list the entries with just one space after each comma as in the code extract below.

The code extract shows how four scannos/per line should be arranged to avoid the _flake8_ error. The new code will, of course, have more scannos per line up to the maximum line length that flake8 allows.

Note that the list assigment has to be bracketed by "# fmt: on/# fmt: off" lines to prevent _black_ reformatting the list entries to one per line.
```
# fmt: off
scannos_misspelled = [
    "1gth",  "1lth", "1oth", "1Oth", 
    "2gth", "2ist", "2lst", "oa", 
    "po", "2Oth", "2OTH", "2oth", 
    "3lst",  "3Oth", "3oth", "abead",
    "ablc", "Abont", "abovc", "abscnt",
    "awav"
]
# fmt: on
print(scannos_misspelled)
```

Reply:
Yes, please, let's use the ^^ format and just have black ignore the lines. The way it is now is going to be error prone I think.

Reply:
It's fixed now in my version of PPtxt. The 3,451 scanno words are assigned in the conventional way (see above) to the scannos_misspelled list in just 390 or so lines. Extending the list should be less prone to error and it doesn't matter if duplicates are present in the list. The few lines of code that build the scannos dictionary from that list ignores any duplicates.